test/grouping.jl

```diff
@@ -4378,4 +4378,132 @@ end
     @test_throws ArgumentError combine(gdf, :a => (x -> [Dict('x' => 1)]) => AsTable)
 end
 
+@testset "sorting API" begin
+    # simple tests
+    df = DataFrame(x=["b", "c", "b", "a", "c"])
+    @test getindex.(keys(groupby(df, :x)), 1) == ["b", "c", "a"]
```

Comment:
I think these can use `only` instead of `getindex(_, 1)`.

Reply:
indeed it could. I initially had this test implemented on groups not on keys and using `getindex` works on both.